vector/src/main/AndroidManifest.xml

```diff
@@ -204,6 +204,14 @@
 
                 <data android:scheme="https" />
                 <data android:host="riot.im" />
+            </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
                 <data android:host="app.element.io" />
```

Comment:
> LGTM, hence I do not see the PR to update element.io website anymore (to see if it's been merged and deployed).

This is now unblocked as I've added https://element.io/.well-known/assetlinks.json via https://github.com/vector-im/element.io-website/pull/96

Reply:
thanks for the update! will rebase this PR